playbooks/aws/openshift-cluster/tasks/launch_instances.yml

```diff
@@ -4,7 +4,9 @@
     docker_vol_ephemeral: "{{ lookup('env', 'os_docker_vol_ephemeral') | default(false, true) }}"
     env: "{{ cluster }}"
     env_host_type: "{{ cluster }}-openshift-{{ type }}"
+    env_node_type: "{{ cluster }}-openshift-{{ g_node_type }}"
```

Comment:
we're going to be removing env_host_type as that pattern doesn't scale.

So I'd prefer not having a new env_ that we'll eventually have to remove. We'll instead use Ansible host pattern expressions when selecting hosts.